hsctl

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
```

Comment:
True, this does lend some flexibility across differing systems.

May be worth the trade off of having explicit control over which bash is called as the chances of someone writing an exploit named bash and injecting it into the path is slim.

I could look at doing this across the project in the dozen-ish scripts that would use it.